lib/datasource/docker/index.spec.ts

```diff
@@ -458,6 +458,34 @@ describe(getName(), () => {
       expect(res.releases).toHaveLength(1);
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('uses quay registry', async () => {
+      const tags = [{ name: '1.0.0' }];
+      httpMock
+        .scope('https://quay.io')
+        .get('/v2/')
+        .reply(200, '', {})
+        .get('/api/v1/repository/node/tag/?limit=10000&page=1')
+        .reply(
+          200,
+          { tags, has_additional: false },
+          {
+            link:
+              '<https://api.github.com/user/9287/repos?page=3&per_page=100>; rel="next", ',
```

Comment:
Fix this url, should point to quay i think.

Reply:
Link is not returned in the headers in this test:
```
✗ curl -I "https://quay.io/api/v1/repository/bitnami/sealed-secrets-controller/tag/?limit=10000&page=1&onlyActiveTags=true"
HTTP/2 200 
server: nginx/1.12.1
date: Wed, 05 May 2021 16:56:12 GMT
content-type: application/json
content-length: 39382
access-control-allow-origin: *
access-control-allow-methods: HEAD, OPTIONS, GET
access-control-max-age: 21600
access-control-allow-headers: AUTHORIZATION, CONTENT-TYPE, X-REQUESTED-WITH
x-frame-options: DENY
strict-transport-security: max-age=63072000; preload
```
so removed it completely.